### PR TITLE
WEBEU2 - JavaScript II - Wasiu Idowu 🇳🇬 

### PR DESCRIPTION
**Pull request by Hoxtygen**

### CODE REVIEW

File: assignments/array-methods.js [hunk 1]
```diff
@@ -56,28 +56,60 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
...
+    fullName.push(`${runner.first_name} ${runner.last_name}`);
+})
 console.log(fullName);
+//console.table(fullName);
 
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
 let allCaps = [];
-console.log(allCaps); 
+runners.map(runner => {
```
Comment (maximesalomon) [inline assignments/array-methods.js:68]: If you want, you can directly do const allCaps = runners ...

In that case, you won't have to push it to allCaps

File: assignments/array-methods.js [hunk 2]
```diff
@@ -56,28 +56,60 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
...
+})
+//console.table(highDonor);
+console.log(largeSize);s
 
-// Problem 3
\ No newline at end of file
+// Problem 3
+runners.forEach(runner => {
+    if (runner.shirt_size == "XS") {
+        runner.message = "Due to production reason, we could not get the appropriate size for you right now, please bear with us as we sort things out";
```
Comment (maximesalomon) [inline assignments/array-methods.js:111]: Noiiice :)

**Review by maximesalomon — COMMENTED** [review]
Great job on this project Wasiu!

File: assignments/array-methods.js (hunk 1 above)
Comment (Hoxtygen) [reply assignments/array-methods.js:68]: copy that

**Pull request merged**